docs/api/js/accounts-l1-l2.md

```diff
@@ -155,6 +155,24 @@ async getBaseCost(params: {
 | params.gasPrice (optional)  | The gas price of the L1 transaction that will send the request for an execute call.                    |
 | returns                     | The base cost in ETH for requesting the contract call.                                                 |
 
+## Claim Failed Deposit
+
+The `claimFailedDeposit` method withdraws funds from the initiated deposit, which failed when finalizing on L2.  
+If the deposit L2 transaction has failed it sends an L1 transaction calling claimFailedDeposit method of the L1 bridge, which results in returning L1 tokens back to the depositor, otherwise throws the error.
+
+### Input Parameters
+
+| Parameter |       Type          | Description                                                                               |
+| --------- |       ------------  | ----------------------------------------------------------------------------------------- |
+| depositHash       | `bytes32`   | 2 transaction hash of the failed deposit.                                    |
+| _depositSender    | `address`   | The address of the deposit initiator. |
+| _l1Token          | `address`   | The address of the deposited L1 ERC20 token.  |
+| _l2TxHash         | `bytes32`   | The L2 transaction hash of the failed deposit finalization.    |
+| _l2BlockNumber    | `uint256`   | The L2 block number where the deposit finalization was processed.   |
+| _l2MessageIndex   | `uint256`   | The position in the L2 logs Merkle tree of the l2Log that was sent with the message.                    |
+| _l2TxNumberInBlock| `uint16`    | The L2 transaction number in a block, in which the log was sent. |
+| _merkleProof      | `bytes32[]` | The Merkle proof of the processing L1 -> L2 transaction with deposit finalization |
```

Comment:
There is only `depositHash` parameter for the `claimFailedDeposit` SDK method, others should be removed, also, please add a method declaration for this method, just like for other methods on this page. Here is a declaration:
```
async claimFailedDeposit(depositHash: BytesLike): Promise<ethers.ContractTransaction>
```